RecoLocalMuon/GEMSegment/plugins/SealModule.cc

```diff
@@ -0,0 +1,9 @@
+#include "FWCore/PluginManager/interface/ModuleDef.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include <RecoLocalMuon/GEMSegment/plugins/ME0SegmentProducer.h>
+#include <RecoLocalMuon/GEMSegment/plugins/ME0SegmentBuilderPluginFactory.h>
+#include <RecoLocalMuon/GEMSegment/plugins/ME0SegAlgoMM.h>
+
+DEFINE_FWK_MODULE(ME0SegmentProducer);
```

Comment:
i ll do this next time together with moving ME0SegmentProducer.h to the .cc file